Dockerfile

```diff
@@ -250,9 +250,15 @@ ENTRYPOINT ["./entrypoint.sh"]
 # Final image
 FROM base
 
-COPY ./tgi-entrypoint.sh /tgi-entrypoint.sh
-RUN chmod +x /tgi-entrypoint.sh
-
 ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/root/.local/share/uv/python/cpython-3.11.11-linux-x86_64-gnu/lib/"
-ENTRYPOINT ["/tgi-entrypoint.sh"]
-# CMD ["--json-output"]
+ENV UID=1000
+ENV USER=llm
```

Comment:
We should make these `ARG`, so that the user can pass them if they are using another uid. Probably GID should be separately settable?